js/render/draw_circle.js

```diff
@@ -4,6 +4,30 @@ var browser = require('../util/browser.js');
 
 module.exports = drawCircles;
 
+var PROPERTIES = [
+    {
+        styleName: 'circle-color',
+        styleType: 'color',
+        glName: 'a_color',
+        glWidth: 4,
+        glType: '4fv'
+    },
+    {
+        styleName: 'circle-blur',
+        styleType: 'number',
+        glName: 'a_blur',
+        glWidth: 2,
+        glType: '1f'
+    },
+    {
+        styleName: 'circle-radius',
+        styleType: 'number',
+        glName: 'a_size',
+        glWidth: 2,
+        glType: '1f'
+    }
+];
+
```

Comment:
I have mixed feelings about this. I sort of like it but I also feel that it adds another level of abstraction without making things clearer.